locales/pt/validation.php

```diff
@@ -104,10 +104,10 @@
     'prohibited'           => 'O campo :attribute é proibido.',
     'prohibited_if'        => 'O campo :attribute é proibido quando :other é :value.',
     'prohibited_unless'    => 'O campo :attribute é proibido a menos que :other esteja em :values.',
-    'prohibits'            => 'The :attribute field prohibits :other from being present.',
+    'prohibits'            => 'O campo :attribute proíbe :other de estar presente.',
     'regex'                => 'O formato do valor para o campo :attribute é inválido.',
     'required'             => 'É obrigatória a indicação de um valor para o campo :attribute.',
-    'required_array_keys'  => 'The :attribute field must contain entries for: :values',
+    'required_array_keys'  => 'O campo :attribute deve conter entradas para: :values',
```

Comment:
```suggestion
    'required_array_keys'  => 'O campo :attribute deve conter entradas para: :values.',
```
I noticed that the error text in the Laravel repository also [does not have a full stop](https://github.com/laravel/laravel/pull/5798/files) at the end of the sentence, but we try to make it for people :)